packages/gatsby/src/utils/source-nodes.ts

```diff
@@ -68,20 +73,26 @@ function getStaleNodes(state, nodes) {
 /**
  * Find all stale nodes and delete them
  */
-function deleteStaleNodes(state, nodes) {
+function deleteStaleNodes(state: IGatsbyState, nodes: Node[]): void {
   const staleNodes = getStaleNodes(state, nodes)
 
   if (staleNodes.length > 0) {
     staleNodes.forEach(node => deleteNode({ node }))
   }
 }
 
-module.exports = async ({ webhookBody = {}, parentSpan } = {}) => {
+export default async ({
+  webhookBody,
+  parentSpan,
+}: {
+  webhookBody?: unknown
```

Comment:
Since I imagine this could be a string or a parsed JSON object from a webhook call, I wasn't sure how tightly to type it.

Reply:
This is correct